### PR TITLE
38737 add missing dutch translation

### LINKED ISSUE #38737: Missing translations for Dutch (nl)
Hello,

There are some missing translation for Dutch (nl). These should be added in branch 3.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.nl.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Core/Resources/translations/security.nl.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by JohJohan**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38737 
| License       | MIT
| Doc PR        | No, not needed

Update dutch translation

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you for this PR. 

Could I ask @sstok and @stephanvierkant to have a look at this?

**Comment by stephanvierkant** [comment]
[login link -> inloglink](https://github.com/symfony/symfony/issues/38737#issuecomment-716196943)

**Review by sstok — APPROVED** [review]

**Review by stephanvierkant — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Comment by Nyholm** [comment]
Awesome!

File: src/Symfony/Component/Security/Core/Resources/translations/security.nl.xlf [hunk 1]
```diff
@@ -66,6 +66,14 @@
                 <source>Account is locked.</source>
                 <target>Account is geblokkeerd.</target>
             </trans-unit>
+            <trans-unit id="17">
+                <source>Too many failed login attempts, please try again later.</source>
+                <target>Te veel onjuiste inlogpogingen, probeer het later nogmaals.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Core/Resources/translations/security.nl.xlf:71]: on `5.x`, this is translated as `Er waren teveel mislukte inlogpogingen, probeer het later opnieuw.`
Which version is the good one?

Comment (nicolas-grekas) [reply src/Symfony/Component/Security/Core/Resources/translations/security.nl.xlf:71]: Please submit again on branch 3.4 if this needs an update.

Comment (sstok) [reply src/Symfony/Component/Security/Core/Resources/translations/security.nl.xlf:71]: "Er waren teveel" is past tense, either "There were to many". The translation above is the correct one.

Comment (stephanvierkant) [reply src/Symfony/Component/Security/Core/Resources/translations/security.nl.xlf:71]: I agree, "Te veel onjuiste inlogpogingen" is a better translation.

Comment (Nyholm) [reply src/Symfony/Component/Security/Core/Resources/translations/security.nl.xlf:71]: Thank you

**Pull request merged**